wiki/Placeholders.md

```diff
@@ -2977,6 +2977,14 @@ Please see [this discussion][list] for a list of all expansions officially maint
 
 ----
 
+- ### **[LevelledMobs](https://www.spigotmc.org/resources/levelledmobs-for-1-14-x-1-17-x.74304/)**
```

Comment:
Please remove the pointless text, as mentioned in the [README](https://github.com/PlaceholderAPI/PlaceholderAPI/tree/docs/wiki/wiki#special-notes)

```suggestion
- ### **[LevelledMobs](https://www.spigotmc.org/resources/74304/)**
```